src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
+$calendar-padding: 10px;
+$element-day-size: 100px;
+$space-between-days: 1px;
+$max-width-calendar: (
+  7 * $element-day-size + 6 * $space-between-days + 2 * $calendar-padding
+);
+$days: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+
+  font-family: Arial, 'Helvetica Neue', Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  box-sizing: border-box;
+  padding: $calendar-padding;
+  display: flex;
+  width: $max-width-calendar;
+  flex-wrap: wrap;
+  gap: $space-between-days;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: $element-day-size;
+    height: $element-day-size;
+    background-color: #eee;
+    border: 1px solid #000;
+    transition:
+      transform 0.5s,
+      background-color 0.5s;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translateY(-20px);
+    }
+  }
+
+  @each $day, $count in $days {
+    &--start-day-#{$day} {
+      :first-child {
+        margin-left: (
+          ($element-day-size * $count) + ($space-between-days * $count)
+        );
```

Comment:
To improve readability, consider using a single variable to calculate the margin-left value instead of performing the calculation inline. This will make it easier to understand and maintain the code. For example, you could define a variable at the top of your SCSS file that calculates the offset based on the day and use it here.